conf/logback.xml

```diff
@@ -2,11 +2,11 @@
     <contextName>editorial-production-metrics</contextName>
 
     <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>${application.home}/logs/application.log</file>
+        <file>${application.home:-.}/logs/application.log</file>
 
         <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
             <fileNamePattern>
-                ${application.home}/logs/application.log.%i
+                ${application.home:-.}/logs/application.log.%i
```

Comment:
not seen this syntax before, what does it do?

Reply:
I wasn't familiar with it either, but noticed it in other projects logback.xml while trying to resolve the error "application.home_IS_UNDEFINED" from logs.gutools.co.uk.

It provides an alternative value if application.home is undefined.  See https://logback.qos.ch/manual/configuration.html#defaultValuesForVariables